admin_tests/pre_reg/test_views.py

```diff
@@ -418,3 +432,11 @@ def test_wrong_provider_odd(self):
         with nt.assert_raises(Http404):
             for item in get_metadata_files(self.draft):
                 pass
+
+    def test_delete_pre_submit_draft_does_not_change_checkouts(self):
```

Comment:
I'm not thrilled adding this test here, but there are no other prereg file tests and it requires all of the setup that occurs here. Could possible move to osf_tests/test_prereg, but most of that is for the views.